docs/app/components/navigation-bar/navigation-bar.component.html

```diff
@@ -14,6 +14,8 @@
 
         <div class="navigation-bar-spacer"></div>
 
+        <uxd-theme-selector *ngIf="themes" title="Theme" [themes]="themes"></uxd-theme-selector>
```

Comment:
We might want to change the Input to something other than `title` as this will result in a browser tooltip when hovering on the theme switcher.

Reply:
Yes, good point, thank you!